postreise/analyze/generation/curtailment.py

```diff
@@ -128,17 +129,17 @@ def calculate_curtailment_percentage_by_resources(scenario, resources=None):
 def calculate_curtailment_time_series_by_areas_and_resources(
     scenario, areas=None, resources=None
 ):
-    """Calculate a time series of curtailment for a set of valid areas and resources.
+    """Calculate hourly curtailment of each generator located in area(s) area and
+    fueled by resource(s)
```

Comment:
`resources(s)` -> `resources(s).`

Reply:
should it be areas(s) in place of area(s)?

Reply:
Sorry, I mis-typed. We want the period at the end of the sentence.

Reply:
Done